dotnet/src/Connectors/Connectors.Google.UnitTests/Core/Gemini/GeminiFunctionTests.cs

```diff
@@ -95,8 +95,8 @@ public void ItCanConvertToFunctionDefinitionsWithParameterTypesAndReturnParamete
                                          {   "type": "object",
                                          "required": ["param1", "param2"],
                                          "properties": {
-                                         "param1": { "type": "string", "description": "String param 1" },
-                                         "param2": { "type": "integer", "description": "Int param 2" }   } }
+                                         "param1": { "description": "String param 1", "type": "string" },
```

Comment:
The changes in the assertion reflect the fact that the new component inserts the `description` keyword at the start of the schema object. This reflection [convention from JSON schema](https://json-schema.org/learn/getting-started-step-by-step#create-a-schema-definition) even though they are equivalent -- property order isn't significant in JSON.